kubeflow/apps/pipeline/Kptfile

```diff
@@ -26,3 +26,88 @@ openAPI:
             ref: '#/definitions/io.k8s.cli.setters.name'
           - marker: ${gcloud.core.project}
             ref: '#/definitions/io.k8s.cli.setters.gcloud.core.project'
+    io.k8s.cli.substitutions.mysql-name:
+      x-k8s-cli:
+        substitution:
+          name: mysql-name
+          pattern: ${name}-kfp
+          values:
+          - marker: ${name}
+            ref: '#/definitions/io.k8s.cli.setters.name'
+    io.k8s.cli.setters.gcloud.compute.region:
+      x-k8s-cli:
+        setter:
+          name: gcloud.compute.region
+          value: asia-east1
```

Comment:
Should we use a placeholder value instead of asia-east1 here? Or use gcloud.compute.zone instead for `io.k8s.cli.substitutions.cloudsql-instance-connection-name`?

In full kubeflow deployment, we are not setting `gcloud.compute.region`, we set `gcloud.compute.zone` only.

Reply:
this has to be a region